script/prerender/manage.ts

```diff
@@ -0,0 +1,397 @@
+// tslint:disable:no-console
+
+/*
+  Manages a fleet of spot instances
+  Creates a cloudformation stack with a spot fleet and SQS queues, preloads all the book ToCs,
+  queues up each page in the queue to be prerendered, and deletes the stack at the end
+*/
+
+import './setup';
```

Comment:
if this file isn't rendering any react code it probably doesn't need this setup thing

Reply:
Just removing it leads to an error, but I figured it's actually contentPages that's loading the React stuff so I moved this import there and it works.